content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -459,6 +460,186 @@ With the above configuration, pods can use `namespaces` and `namespaceSelector`
 if the namespace where they are created have a resource quota object with
 `CrossNamespacePodAffinity` scope and a hard limit greater than or equal to the number of pods using those fields.
 
+### Resource Quota Per VolumeAttributesClass
+
+{{< feature-state feature_gate_name="VolumeAttributesClass" >}}
+
+PersistentVolumeClaims can be created with a specific [volume attributes class](/docs/concepts/storage/volume-attributes-classes/), and might be modified after creation. You can control a PVC's consumption of storage resources based on the associated volume attributes classes, by using the `scopeSelector` field in the quota spec.
+
+The PVC references the associated volume attributes class by the following fields:
+
+* `spec.volumeAttributesClassName`
```

Comment:
Can we add an explanation that these three fields will be all calculated against the quota?

Reply:
I meant giving an example like here: https://github.com/kubernetes/enhancements/pull/5028#issuecomment-2581683871

Reply:
@sunnylovestiramisu added in the below example.